VMAccess/vmaccess.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+COMMAND="./vmaccess.py"
+PYTHON=""
+
+USAGE="$(basename "$0") [-h] [-i|--install] [-u|--uninstall] [-d|--disable] [-e|--enable] [-p|--update] -- program to invoke the extension script
```

Comment:
consider moving the description of the script "program to invoke the extension script" to another line to separate it from the syntax